domain-ee/ee-ep-merge-app/src/python_src/service/ep_merge_machine.py

```diff
@@ -23,6 +23,16 @@
 from service.job_store import JOB_STORE
 from statemachine import State, StateMachine
 from util.contentions_util import ContentionsUtil
+from util.metric_logger import distribution, increment
+
+ERROR_STATES_TO_LOG_METRICS = [JobState.RUNNING_CANCEL_EP400_CLAIM, JobState.RUNNING_ADD_CLAIM_NOTE_TO_EP400]
```

Comment:
Why only log metrics for these 2 error states?

Reply:
So the metrics that are only logged on those states are the new contentions metric for distribution related to 
this [comment](https://github.com/department-of-veterans-affairs/abd-vro/issues/2055#issuecomment-1922081295)

`ep_merge.job.skipped_merge` and `ep_merge.job.new_contentions` should be logged if we get to the point where we cancel the job because if it reached that point then we would have past the point where any new contentions would have been successfully merged to the pending EP. But actually, I think i need to add in all the potential failed steps associated with the jobs states to cancel the ep400 or add claim note. If those steps fail, then we should still log the metric as well.